packages/models-library/src/models_library/services.py

```diff
@@ -275,6 +275,13 @@ class Config(ServiceProperty.Config):
                     "type": "number",
                     "unit": "second",
                 },
+                # folder
```

Comment:
this comment is confusing. there is no folder as far as I know... or is there now a new type of entry?
Also the comment ```# latest:``` above, what does it mean?